docs/top.rst

```diff
@@ -54,6 +55,9 @@ This level enables fully connected multi-layer perceptron.
    nnvm.symbol.softmax
    nnvm.symbol.log_softmax
    nnvm.symbol.pad
+   nnvm.symbol.assign
```

Comment:
for now let us not include assign in the graph, we can use it to get the update and do assignments in the outside